public/components/security/roles-mapping/roles-mapping-create.tsx

```diff
@@ -0,0 +1,139 @@
+
+import React, { useState, useEffect } from 'react';
+import {
+    EuiToolTip,
+    EuiButtonIcon,
+    EuiButton,
+    EuiTitle,
+    EuiFlyout,
+    EuiFlyoutHeader,
+    EuiFlyoutBody,
+    EuiForm,
+    EuiFormRow,
+    EuiSpacer,
+    EuiFlexGroup,
+    EuiFlexItem,
+    EuiComboBox,
+    EuiFieldText,
+} from '@elastic/eui';
+import { WzRequest } from '../../../react-services/wz-request';
+import { ErrorHandler } from '../../../react-services/error-handler';
+import { RuleEditor } from './rule-editor';
+
+export const RolesMappingCreate = ({ closeFlyout, rolesEquivalences, roles }) => {
+    const [selectedRoles, setSelectedRoles] = useState([]);
+    const [ruleName, setRuleName] = useState("");
+    const [isLoading, setIsLoading] = useState(false);
+
+    useEffect(() => {
+        initData();
+    }, []);
+
+    const initData = () => {
+    }
+
+    const getEquivalences = (roles) => {
+        const list = roles.map(item => {
+            return { label: rolesEquivalences[item], id: item }
+        });
+        return list;
+    };
+
+    const getRolesList = (roles) => {
+        const list = roles.map(item => {
+            return { label: rolesEquivalences[item.id], id: item.id }
+        });
+        return list;
+    };
+
+    const onChangeComboBox = (roles) => {
```

Comment:
If you want to change the name of the setSelectedRoles function, rename it in the useState assignment. Although renaming doesn't really add semantics to the code, in your component when you assign as a value the function setSelectedRoles to the prop 'onChange' you know that when the state of the component changes it will select a role.